requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+flake8
+flake8-bugbear
```

Comment:
while it is true that I love flake8 with all of my heart, I don't think we need the extensions for this. `flake8` alone will be plenty.

Reply:
Removed.